### ISSUE TITLE
Exception when running specifically named tests in DescribeSpec

### ISSUE THREAD

**Issue by xcq1** [kotest/kotest#5551]
If I try to run the `describe` or the `it` of a DescribeSpec I'm getting a weird exception:
```
Exception in thread "main" java.lang.RuntimeException: Could not detect spec class style for supertypes [class io.kotlintest.specs.DescribeSpec]
	at io.kotlintest.runner.console.SpecAwareTestFilter.<init>(SpecAwareTestFilter.kt:27)
	at io.kotlintest.runner.console.KotlinTestConsoleRunner.execute(KotlinTestConsoleRunner.kt:12)
	at io.kotlintest.runner.console.LauncherKt.main(launcher.kt:17)
```

I narrowed it down to the naming of the test case and came up with this rather odd minimal example:
```
class Example : DescribeSpec({
    describe("withoutTLSValidation") {
        it("allowsto") {
        }
    }
})
```
As soon as I change the strings a little, it does work. Does not seem to be related to caching either as I can paste this in a brand new project and the same exception occurs.

Plugin: 3.3.0.8 
kttest: 3.3.0-RC3

**Comment by sksamuel** [comment]
I think it's a known bug for describe and expect (in that I forgot to
implement those two).
Fix coming in 3.3.0.9

**Comment by sksamuel** [comment]
3.3.0.10 is now released compiled against KotlinTest 3.3.0
This is the first "release" version of both.
Please make sure you have upgraded both and you should be good to go!
(It might take a day before Jetbrains approve the plugin release)
